includes/templates/template_default/templates/tpl_modules_address_book_details.php

```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types=1);
```

Comment:
Why?

Reply:
My mistake, I put that in my files as I go along.